Parser/asdl_c.py

```diff
@@ -1860,16 +1871,20 @@ def visitModule(self, mod):
 
 static int add_ast_fields(struct ast_state *state)
 {
-    PyObject *empty_tuple;
+    PyObject *empty_tuple, *empty_dict;
     empty_tuple = PyTuple_New(0);
-    if (!empty_tuple ||
+    empty_dict = PyDict_New();
```

Comment:
I think this is technically unsafe: if PyTuple_New raised an exception, we can't generally call other C API functions with an exception set. Safer to have a separate if statement for the dict.